### ISSUE TITLE
Critical Accessibility Issue: Not announcing Current Page for nav-link

### ISSUE THREAD

**Issue by fortunatomaldonado** [liferay/clay#5002]
## What are the steps to reproduce?
1. Use a Screen Reader like JAWS
2. Start Liferay and add a Calendar to a page
3. Tab through until you reach the nav-link Calendar

This can also be seen in https://clayui.com/docs/components/navigation-bar/markup.html#clay-sidebar
Current Page is never announced by Screen Reader

## What is the expected result?
Expected:
The Screen Reader should announces Current Page for the nav-link Calendar that is currently open

Actual: 
The Screen Reader does not announce Current Page for the nav-link Calendar that is currently open

## Environment

| Tech  | Version |
| ----- | ------- |
| Clay  | v3.65.2  |
| React |         |

**Comment by fortunatomaldonado** [comment]
Hello @matuzalemsteles, thank you for taking a look at this.
I believe Lianne was able to provide a fix here: https://github.com/fortunatomaldonado/clay/pull/1

I will forward the fix shortly.

**Comment by matuzalemsteles** [comment]
Oh that's great. No problem, this can be forwarded. After we fix it on our side I'll take care of removing it there so we can leave it on Clay's side, thanks!

**Comment by matuzalemsteles** [comment]
Oh I figured that out now 🤦‍♂️, I thought this was going to be for DXP! Yes, you can send a fix to Clay, thanks!

**Comment by wanderlast** [comment]
Sent a fix here: https://github.com/liferay/clay/pull/5003

Let me know if anything needs adjusting!
